.github/ISSUE_TEMPLATE/report-bug.md

```diff
@@ -1,10 +1,13 @@
 ---
 name: Report Bug
 about: Report bug to OpenLab.
+title: ''
+labels: bug
+assignees: ''
 
 ---
 
-Let's make things better!
+Thank you for reporting bug!
```

Comment:
"reporting a bug" or "reporting this bug" would be more grammatically correct, but I don't think it's a big deal.